code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm

```diff
@@ -36,3 +36,7 @@
 		adjustBruteLoss(10)
 	else if(bodytemperature > maxbodytemp)
 		adjustBruteLoss(15)
+
+/mob/living/simple_animal/hostile/retaliate/clown/attack_hand(mob/living/carbon/human/M)

```

Comment:
atm it only works on help intent and does nothing on disarm/harm, making it inconsistent with human clowns